### PR TITLE
Add inference into CI v2

### PR DESCRIPTION
**Pull request by mikolajzalewski**
1. Added 'inference' directory containing 3 files:
 -infer_assertions.sh - script used for preprocessing and performing reasoning on the ontology
 -ofn_to_ttl.sh - script used for translating OWL funcional syntax to Turtle syntax
 -skos_patch.ttl - Turtle file containing some of SKOS Object Properties. Used in preprocessing of the ontology

2. Added 3 steps in 'pre-release' pipeline including:
 -downloading robot.jar file
 -installing Java
 -inferring assertions

As a result, additional inferred_assertions.ttl file with all neccessary inferences is created.

### CODE REVIEW

File: inference/infer_assertions.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin bash
```
Comment (Ostrzyciel) [inline inference/infer_assertions.sh:1]: Oh my, I didn't even notice the bash files in the previous PR! ...my bad. Anyway – no, bash files for CI should not be committed to the main repo, that's not a good practice. So, sorry, but I don't see any way you can avoid modifying the ci-worker. Please:
- Move the bash files to the ci-worker repo (create a new directory for inference, perhaps)
- Modify the ci-worker's Dockerfile to add a step where you download ROBOT's JAR via wget
- Update this pull request accordingly

...and setting up Java can stay as a CI step here, it's acceptable for now.

File: inference/skos_patch.ttl [hunk 2]
```diff
@@ -0,0 +1,134 @@
+#################################################################
```
Comment (Ostrzyciel) [inline inference/skos_patch.ttl:1]: Have you tried adding an import to SKOS in the header instead? Having a patch like this is what's professionally called "ugly" :)

**Review by Ostrzyciel — CHANGES_REQUESTED** [review]

**Comment by Ostrzyciel** [comment]
You didn't have to submit a whole new PR – this way the history of the previous comments was erased. Next time, please use git's rebase feature.

File: inference/skos_patch.ttl (hunk 2 above)
Comment (wojtas000) [reply inference/skos_patch.ttl:1]: SKOS import is done automatically, in fact this file is used to unmerge unneccesary properties. Please read our final report to find more details on this.

Comment (Ostrzyciel) [reply inference/skos_patch.ttl:1]: Ah! Okay. Then please document what this file is for in the repo. Something like `inference/README.md`.

**Review by Ostrzyciel — APPROVED** [review]

**Pull request merged**